pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTMethodDeclaration.java

```diff
@@ -139,21 +139,11 @@ public ASTDefaultValue getDefaultClause() {
         return AstImplUtil.getChildAs(this, getNumChildren() - 1, ASTDefaultValue.class);
     }
 
-    /**
-     * Returns the result type node of the method.
-     *
-     * @deprecated todo When removed from java-grammar, rename the other to this good name
-     */
-    @Deprecated
-    public ASTResultType getResultType() {
-        return getFirstChildOfType(ASTResultType.class);
-    }
-
     /**
      * Returns the result type node of the method. This may be a {@link ASTVoidType}.
      */
     public @NonNull ASTType getResultTypeNode() {
```

Comment:
```suggestion
    public @NonNull ASTType getResultTypeNode() { // TODO rename to getResultType()
```

Let's just keep the TODO to not forget it - I'd do it later not to add more changes in this PR.